### PR TITLE
WDP200303-11

### PR DESCRIPTION
**Pull request by jurecki**
handle favorite button has been added:
 - add action to productRedux 
 - add dispatcher to NewFurnitureContainer

### CODE REVIEW

**Comment by Varenthein** [comment]
Please add a proper description.

**Comment by Varenthein** [comment]
It this task ready? It doesn't look like that.

**Comment by jurecki** [comment]
Please check it again, I resolved conflicts, 
Now, everything will be ok.

File: src/components/common/Button/Button.js [hunk 1]
```diff
@@ -19,7 +28,12 @@ const Button = ({ children, variant, noHover, className: propClassName, ...props
   }
 
   return (
-    <Comp href='#' {...props} className={classes.join(' ')}>
+    <Comp
+      href='#'
+      {...props}
```
Comment (Varenthein) [inline src/components/common/Button/Button.js]: When you add `href=#` to the link, every click causes scroll to the top of the page.  You don't have to `set this attr at all.

Comment (jurecki) [reply src/components/common/Button/Button.js]: I don't think that is good idea. Button is universal component, href would be necessary in other project. I add props href with "false". Now will be ok.

Comment (Varenthein) [reply src/components/common/Button/Button.js]: Okay, you've got the point, but... we can't use `false` as `href` attr. Just look to the console. There are many warnings now about this.

Comment (jurecki) [reply src/components/common/Button/Button.js]: I used e.preventDefault() to remove scroll page

**Pull request merged**